pkg/registry/admissionregistration/rest/storage_apiserver.go

```diff
@@ -29,31 +29,33 @@ import (
 	validatingwebhookconfigurationstorage "k8s.io/kubernetes/pkg/registry/admissionregistration/validatingwebhookconfiguration/storage"
 )
 
-type RESTStorageProvider struct{}
+// StorageProvider is a REST storage provider for storage_apiserver
```

Comment:
`storage_apiserver` is not a user-facing term.

How about "StorageProvider is a REST storage provider for the admissionregistration API group."

Reply:
Also please add a "." at the end of each comment, see https://blog.golang.org/godoc.

Reply:
I get it , Thank  you so much